### DISCUSSION TITLE
Can It have a Standalone Ver.  exe for CPU  inpaintMesh

### DISCUSSION THREAD

**Discussion by LanceGao97** [thygate/stable-diffusion-webui-depthmap-script#253] [Ideas]
Hi,  can it  have a standalone version , for 64bit windows7,   my old computer still have a nice CPU i7 7900,  but its GPU don't support stablediffusion,   ， while  this inpainting Mesh is really slow and it is using CPU only,  the GPU is 0% working,   If this can be possible, 
batch render that inpaint Mesh would be available,   while  new computer can be used to play around in Stable Diffusion.

**Comment by thygate** [comment]
you can still install A1111's without gpu support and just run the depth extension from it.. If you want a stand alone implementation, just use the original repo of the process you want ..

**Reply by LanceGao97** [reply]
Thanks thygate  ， thats really helpful.    no more info, everythiing fine.

**Reply by LanceGao97** [reply]
> you can still install A1111's without gpu support and ju  
https://www.youtube.com/shorts/8X5CeEeq4NA  
Hi  thygate  thanks for making such a wonderful extenssion,  after reading so many discussions, about docker , jubiter EGL, backend,
I don't know any of those words.           , I  still can't make the videos on autoDL or Google Colab.  

Please add a function of " batch input .PLY files to render MP4 videos" 
cause currently  , the online server autoDL and GoogleColab all errors after making PLY and OBJ,  

 I can only download those PLY and render videos on my own computer,  which means hand input ,  if I wanna make serveral videos, 
I m gonna hand input those locations of PLY files for hours.
For now I got 60 PLY files , and Im gonna make more to make several videos,    I think that will kiil me if I hand input those PLY file directories.

**Comment by sonakravtsenko3** [comment]
@LanceGao97 You may find #290 useful, it allows you to not use Automatic1111 and still have the ability to do stuff.
